curriculum/challenges/english/16-the-odin-project/top-learn-to-solve-problems-and-understand-errors/learn-to-solve-problems-and-understand-errors-question-e.md

```diff
@@ -0,0 +1,57 @@
+---
+id: 66581a7cb1eb2281159492fe
+title: Learn to Solve Problems and Understand Errors Question E
+challengeType: 15
+dashedName: learn-to-solve-problems-and-understand-errors-question-e
+---
+
+# --description--
+
+Let’s assume you have written the following code:
+
+```javascript
+const a = "Hello";
+const b = "World";
+
+console.log(c);
+```
+
+This code will run, but it will generate an error. In technical terms, this is called "throwing" an error. The first part of an error displays the type of error. This provides the first clue as to what you're dealing with. You'll learn more about the different error types later in the lesson. In this example, you have a `ReferenceError`.
```

Comment:
```suggestion
This code will run, but it will generate an error. In technical terms, this is called <dfn>throwing</dfn> an error. The first part of an error displays the type of error. This provides the first clue as to what you're dealing with. You'll learn more about the different error types later in the lesson. In this example, you have a `ReferenceError`.
```